src/components/SettingsList/SettingsList.tsx

```diff
@@ -87,13 +87,15 @@ const SettingsList = () => {
               </label>
             </div>
           </div>
-          <div className='settings-item'>
+          <div className={`settings-item ${unsavedSettings.playerAccess.canModifyAllTurns ? 'settings-item-disabled' : ''}`}
```

Comment:
I've been using clsx to manage multiple classnames when they need to be conditional. Perhaps move to use that? Looks a bit cleaner

```suggestion
          <div className={clsx('settings-item', { settings-item-disabled: unsavedSettings.playerAccess.canModifyAllTurns})}
```